lib/filesystem/FileStream.cpp

```diff
@@ -45,11 +45,11 @@ voidpf call_zopen64 (const zlib_filefunc64_32_def* pfilefunc,const void*filename
 
 long call_zseek64(const zlib_filefunc64_32_def* pfilefunc, voidpf filestream, ZPOS64_T offset, int origin)
 {
-	if (pfilefunc->zfile_func64.zseek64_file != NULL)
+	if (pfilefunc->zfile_func64.zseek64_file != nullptr)
 		return (*(pfilefunc->zfile_func64.zseek64_file)) (pfilefunc->zfile_func64.opaque, filestream, offset, origin);
 	else
 	{
-		uLong offsetTruncated = (uLong)offset;
+		auto offsetTruncated = (uLong)offset;
```

Comment:
Looks like auto-upgrade have not replaced c-style casts in this file for some reason?
Not the only case.

Or is file too messy for clang somehow?

Reply:
I just skipped it because this was pure C, so, just for purity.
Passed this file also.